Charts/Core/vtkChartXY.cxx

```diff
@@ -1608,6 +1631,69 @@ bool vtkChartXY::MouseMoveEvent(const vtkContextMouseEvent& mouse)
       this->Scene->SetDirty(true);
     }
   }
+  else if (mouse.GetButton() == this->Actions.ClickAndDrag())
+  {
+    if (this->DragPoint)
+    {
+      // Iterate through each corner, and check for a nearby point
+      for (size_t i = 0; i < this->ChartPrivate->PlotCorners.size(); ++i)
+      {
+        int items = static_cast<int>(this->ChartPrivate->PlotCorners[i]
+                                     ->GetNumberOfItems());
+        if (items)
+        {
+          vtkVector2f position;
+          vtkTransform2D* transform =
+              this->ChartPrivate->PlotCorners[i]->GetTransform();
+          transform->InverseTransformPoints(mouse.GetPos().GetData(),
+                                            position.GetData(), 1);
+          for (int j = 0; j < items; ++j)
+          {
+            vtkPlot* plot = vtkPlot::SafeDownCast(this->ChartPrivate->
+                                                  PlotCorners[i]->GetItem(j));
+            if (!plot)
+            {
+              continue;
+            }
+            if (plot->IsA("vtkPlotBar"))
+            {
+              continue;
+            }
```

Comment:
```C++
if (!plot || plot->IsA("vtkPlotBar")) {
    continue;
}
```

and delete these two if above. this is more straight forward. So... the second part of the expression (```plot->IsA("vtkPlotBar")```) will be evaluate only if ```!plot == false```